run-ci.sh

```diff
@@ -13,9 +13,6 @@ echo "EMACS = $EMACS"
 echo
 
 changed_recipes=$(echo "$CHANGED_FILES" | (grep -Po '(?<=^recipes/)[a-z0-9].*' || true))
```

Comment:
Would it be so bad to just always include `kanban` and `magit`? or maybe even `evil`?

Reply:
I thought about this, but I had the concern that contributors would constantly be asking "why is this also checking kanban?  My recipe has nothing to do with kanban."  But I think adding `evil` is a great idea, it's an interesting package with a nontrivial recipe.

Reply:
Should I see the change to add evil? Or did you mean when the tooling changes?